packages/synapse-react-client/src/components/dataaccess/AccessApprovalsTable.tsx

```diff
@@ -140,13 +140,13 @@ export const AccessApprovalsTable: React.FunctionComponent<
           No Results
         </Typography>
       )}
-      {!hasNextPage ? (
-        ''
-      ) : (
+      {hasNextPage && (
         <Button
           variant="outlined"
           color="primary"
-          onClick={() => fetchNextPage}
+          onClick={() => {
+            fetchNextPage()
+          }}
```

Comment:
`fetchNextPage` was never called, so future pages of data could never be fetched